packages/vscode-extension/WHATISNEW.md

```diff
@@ -1,22 +1,41 @@
+- [3.0.0 - Nov 29 2021](#300---nov-29-2021)
 - [2.10.0 - Nov 15 2021](#2100---nov-15-2021)
```

Comment:
Looks the section `2.10.0 - Nov 15 2021` is deleted and the corresponding bookmark is invalid?

Reply:
Should add it back. Let me fix it.